src/demo/java/com/demo/amplitude/DemoController.java

```diff
@@ -0,0 +1,35 @@
+package com.demo.amplitude;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.amplitude.Amplitude;
+import com.amplitude.AmplitudeEventCallback;
+import com.amplitude.AmplitudeLog;
+import com.amplitude.Event;
+
+@RestController
+public class DemoController {
+  @RequestMapping("/")
+  public String index() {
+    Amplitude amplitude = Amplitude.getInstance("INSTANCE_NAME");
+    amplitude.init("8e07b9d451a7d07bd33f6e9ba5870f21");
+    amplitude.logEvent(new Event("Test Event", "test_user_id"));
+    amplitude.setLogMode(AmplitudeLog.LogMode.DEBUG);
+    AmplitudeEventCallback eventCallback =
+        new AmplitudeEventCallback() {
+          @Override
+          public void onEventSent(Event event, int status, String message) {
```

Comment:
@qingzhuozhen I'm re-questioning the name for `onEventSent`. Not sure it is very clear what "sent" means and if it corresponds to logEvent, flush, request sent to server, or server response. Wondering if we can think of something more explicit such as `AmplitudeEventCallback.onServerResponse`

I am also thinking future customers might want additional callbacks, some that might not be per event. Maybe we can rename the interface to be more generic.

```
public interface AmplitudeCallbacks {
    void onLogEventResponse(Event event, int status, String message);
    // Example of potential future callback not related to events
    // void onFlush();
}

amplitude.addCallbacks(new AmplitudeCallbacks() { ... })
```

Reply:
@justin-fiedler I updated the names on class and method. Feel free to take another look. One extra thought about extending this, would each time when we extend AmplitudeCallback interface be a breaking change for upgrade?

Reply:
@qingzhuozhen good point on the breaking change on each update. We could make AmplitudeCallbacks a class instead of an interface and have default noop implementations for all methods. What do you think?

Reply:
@justin-fiedler yeah, I guess switch to a class makes more sense for future change. Personally like interface more but seems that won't be best fit considering future methods. Let me switch it to class then.

Reply:
@justin-fiedler just updated interface to class.